CLAUDE.md

```
@@ -1,120 +1 @@
-
-- `#[tokio::test]` for async, `#[test_log::test(tokio::test)]` for tracing output in tests
-- `NodeConfig::testing()` / `ConsensusConfig::testing()` for test configurations
-- `rstest` for parameterized tests, `proptest` for property-based tests
-- Failure tracking: `cargo xtask test` records failures to `target/nextest-failure-tracking/failures.json`
-- Crypto crates (`irys-vdf`, `irys-packing`, `irys-c`, `sha2`, `openssl`) are always compiled with `opt-level = 3`, even in dev builds
-- Use `irys_testing_utils::TempDirBuilder` for temp dirs in tests — never use `tempfile::tempdir()` or `std::env::temp_dir()` directly, so all test dirs go through `.tmp/`
-
-### Local changes
-
-NOTE: all the above instructions can be overridden and added to as instructed in `./.claude/local/CLAUDE.md` - if it exists, read it and follow any listed instructions before continuing.
+Read and follow @AGENTS.md
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)